eng/Subsets.props

```diff
@@ -233,35 +219,33 @@
   <!-- Installer sets -->
   <ItemGroup Condition="$(_subset.Contains('+corehost+'))">
     <CorehostProjectToBuild Include="$(InstallerProjectRoot)corehost\corehost.proj" SignPhase="Binaries" />
-    <ProjectToBuild Include="@(CorehostProjectToBuild)" BuildInParallel="true" Pack="true" Category="installer" />
+    <ProjectToBuild Include="@(CorehostProjectToBuild)" Pack="true" Category="installer" />
```

Comment:
We could add some indirect dependencies, but this was the easiest way to ensure that the new sfx subset correctly saw updates from the host build with our current build infra. Generally with the subset design, we generally rely on building specific subsets instead of ProjectReferences across subsets, at least for now.